ui/desktop/app/components/credentials-panel/index.hbs

```diff
@@ -24,39 +24,25 @@
 {{#each @credentials as |credential|}}
   {{#if this.isRawApiVisible}}
     <div class='raw-credential-panel-body'>
-      <div class='raw-credential-header'>
-        <div class='raw-credential-title'>
```

Comment:
I think these classes can be deleted now?

Reply:
Do you mean in the CSS file?

Reply:
Yup, I believe they were just used here

Reply:
Done